### PR TITLE
Fix aliases params.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌

I don't understand why the test in PHP 8.0 fails, you can check it @samdark

### CODE REVIEW

**Comment by samdark** [comment]
The error is super-weird. Tests are passing for me locally with PHP 8.0.0alpha3.

File: config/tests.php [hunk 1]
```diff
@@ -18,7 +18,7 @@
     CacheInterface::class => Cache::class,
     LoggerInterface::class => Logger::class,
 
-    Connection::class => static function (ContainerInterface $container) {
+    Connection::class => function (ContainerInterface $container) {
```
Comment (samdark) [inline config/tests.php:21]: We can leave it as is.

File: config/console.php [hunk 2]
```diff
@@ -6,6 +6,9 @@
 
 return [
     Aliases::class => [
-        '@yiisoft/yii/db/migration' => dirname(__DIR__)
+        '__class' => Aliases::class,
+        '__construct()' => [
```
Comment (samdark) [inline config/console.php:10]: That's more explicit but previous code should work as well.

Comment (terabytesoftw) [reply config/console.php:10]: yes, its correct close pr.

**Pull request closed without merging**